### ISSUE TITLE
OIDC compatibility — Self-Issued OpenID Provider

### ISSUE THREAD

**Issue by erlend-sh** [decentralized-identity/didwebvh#128]
Apologies if I’ve made faulty connections, but the way `tdw` is described seems to align very well with this:

https://openid.net/specs/openid-connect-self-issued-v2-1_0.html

https://medium.com/decentralized-identity/where-to-begin-with-oidc-and-siop-7dd186c89796

In other words, could tdw be used as a [[7.2.3.](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html#name-decentralized-identifiers)] Decentralized Identifier in a Self-Issued OpenID Provider?

I guess I’m looking for a modern sequel to the discontinued [did-siop](https://identity.foundation/did-siop/), which I see some of you here co-authored 💖 

The exact fit of SOIP v2 notwithstanding, I think OIDC compatibility makes a lot of sense for an optional (and equally  web-native) ID management layer coupled with `tdw`, or something functionally similar to it.

Further reading on where I’m coming from:

- https://github.com/sebadob/rauthy (very light OIDC server)
- https://socialhub.activitypub.rocks/t/autonomous-identity-for-the-pluriverse-based-on-oauth-oidc/3675?u=erlend_sh
- https://blog.erlend.sh/weird-netizens

**Comment by swcurran** [comment]
Yes, that is correct. I would guess that any SIOP deployments using DIDs are using `did:web` and any use of `did:web` can be improved by transitioning to `did:tdw`.

Interesting stuff!

**Comment by swcurran** [comment]
Closing this issue, as I think it is resolved.
